### PR TITLE
chore: remove deprecated lime APIs

### PR DESCRIPTION
**Pull request by mhamilton723**
## Related Issues/PRs

#xxx

## What changes are proposed in this pull request?

_Briefly describe the changes included in this Pull Request._

## How is this patch tested?

- [ ] I have written tests (not required for typo or doc fix) and confirmed the proposed feature/bug-fix/change works.

## Does this PR change any dependencies?

- [ ] No. You can skip this section.
- [ ] Yes. Make sure the dependencies are resolved correctly, and list changes here.

## Does this PR add a new feature? If so, have you added samples on website?

- [ ] No. You can skip this section.
- [ ] Yes. Make sure you have added samples following below steps.

1. Find the corresponding markdown file for your new feature in `website/docs/documentation` folder.
   Make sure you choose the correct class `estimators/transformers` and namespace.
2. Follow the pattern in markdown file and add another section for your new API, including pyspark, scala (and .NET potentially) samples.
3. Make sure the `DocTable` points to correct API link.
4. Navigate to website folder, and run `yarn run start` to make sure the website renders correctly.
5. Don't forget to add `<!--pytest-codeblocks:cont-->` before each python code blocks to enable auto-tests for python samples.
6. Make sure the `WebsiteSamplesTests` job pass in the pipeline.

AB#1936951

### CODE REVIEW

File: website/docs/documentation/transformers/core/_SuperpixelTransformer.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ spt = (SuperpixelTransformer()
 <TabItem value="scala">
 
 ```scala
+import com.microsoft.azure.synapse.ml.image.SuperpixelTransformer
 import com.microsoft.azure.synapse.ml.lime._
```
Comment (memoryz) [inline website/docs/documentation/transformers/core/_SuperpixelTransformer.md:34]: `com.microsoft.azure.synapse.ml.lime` no longer exists right?

**Review by memoryz — DISMISSED** [review]
One small issue in _SuperpixelTransformer.md. The rest all looks good! Thank you for the cleanup!

File: website/docs/documentation/transformers/core/_SuperpixelTransformer.md (hunk 1 above)
Comment (mhamilton723) [reply website/docs/documentation/transformers/core/_SuperpixelTransformer.md:34]: thanks!

**Review by memoryz — APPROVED** [review]

**Pull request merged**